contracts/src/interfaces/ISubnet.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8.23;
+
+/// @title Subnet interface
+interface ISubnet {
+    /// @notice Checks if the subnet is now bootstrapped
+    function bootstrapped() external view returns(bool);
+}
```

Comment:
Could you elaborate on what responsibility/scope this interface would hold going forward?